source/_patterns/02-molecules/navigation/primary-nav/_primary-nav.scss

```diff
@@ -211,7 +220,8 @@
 
 //mobile
 .jcc-primary-nav {
-  #js-header_menu--mobile {
+  #js-header_menu--mobile,
+  #js-header_menu--mobile-2 {
```

Comment:
Why the -2?

Reply:
@bennlich It has to do with how the component is built and the mobile menu is added to the screen through js. The second mobile menu works on the organism menu while the original works on the molecule.

Reply:
Ah. Do they need different ids?

If they do, would it make sense to name the second one after the component it's used in? E.g. `js-header_menu--mobile--trial-header` ?

Reply:
Good call, fixed!